libs/core/src/lib/message-strip/message-strip.component.html

```diff
@@ -0,0 +1,13 @@
+<button class="fd-message-strip__close"
+        fd-button
+        [compact]="true"
+        [fdType]="'transparent'"
+        *ngIf="dismissible"
+        (click)="dismiss()"
+        [attr.aria-controls]="id"
+        [attr.aria-label]="dismissLabel">
+</button>
+<p class="fd-message-strip__text">
+    <ng-container #container>{{message}}</ng-container>
```

Comment:
What is the purpose of this `<ng-container #container></ng-container>`

Reply:
Removed